src/openscm_units/_unit_registry.py

```diff
@@ -456,25 +459,15 @@ def _load_metric_conversions(self):
 
         This is done only when contexts are needed to avoid reading files on import.
```

Comment:
The comment "to avoid reading files on import" doesn't make sense here anymore.

Reply:
Good spot, will update now